dataland-frontend/tests/component/components/pages/CompanyCockpitPage.cy.ts

```diff
@@ -31,8 +31,9 @@ describe("Component test for the company cockpit", () => {
 
```

Comment:
In dem data owner uploads rights branch, haben Emanuel und ich ein paar Änderungen am Company Cockpit gemacht (Sachen gestreamlined und generalisiert). Da wird es ein paar Merge-Konflikte / Anpassungsbedarf geben.

Reply:
habe ich gemerged